{"timestamp":"2025-07-10T13:30:47.112486","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-07-10T13:30:47.112963","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/bangalore_home_prices_dag.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-07-10T13:30:47.114305","level":"info","event":"Using MLflow tracking URI: http://mlflow:5000","logger":"root"}
{"timestamp":"2025-07-10T13:30:48.844996","level":"info","event":"Starting model training...","logger":"root"}
{"timestamp":"2025-07-10T13:30:50.503430","level":"warning","event":"Unable to find AWS Connection ID 'aws_default', switching to empty.","logger":"airflow.task.hooks.airflow.providers.amazon.aws.hooks.s3.S3Hook"}
{"timestamp":"2025-07-10T13:30:50.504595","level":"info","event":"No connection ID provided. Fallback on boto3 credential strategy (region_name=None). See: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html","logger":"airflow.providers.amazon.aws.hooks.base_aws.BaseSessionFactory"}
{"timestamp":"2025-07-10T13:30:50.856025","level":"info","event":"Found credentials in environment variables.","logger":"botocore.credentials"}
{"timestamp":"2025-07-10T13:30:51.786132","level":"info","event":"Preparing features...","logger":"root"}
{"timestamp":"2025-07-10T13:30:51.974782","level":"info","event":"Splitting data into train and test sets...","logger":"root"}
{"timestamp":"2025-07-10T13:30:51.977409","level":"info","event":"MLflow tracking URI set to: http://mlflow:5000","logger":"root"}
{"timestamp":"2025-07-10T13:30:52.006567","level":"warning","event":"Retrying (Retry(total=6, connect=6, read=7, redirect=7, status=7)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7d9b72d587d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/2.0/mlflow/experiments/get-by-name?experiment_name=bangalore_home_prices","logger":"urllib3.connectionpool"}
{"timestamp":"2025-07-10T13:30:56.874178","level":"warning","event":"Retrying (Retry(total=5, connect=5, read=7, redirect=7, status=7)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7d9b72ce8800>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/2.0/mlflow/experiments/get-by-name?experiment_name=bangalore_home_prices","logger":"urllib3.connectionpool"}
{"timestamp":"2025-07-10T13:31:04.884724","level":"warning","event":"Retrying (Retry(total=4, connect=4, read=7, redirect=7, status=7)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7d9b72ce9160>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/2.0/mlflow/experiments/get-by-name?experiment_name=bangalore_home_prices","logger":"urllib3.connectionpool"}
{"timestamp":"2025-07-10T13:31:21.649384","level":"warning","event":"Retrying (Retry(total=3, connect=3, read=7, redirect=7, status=7)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7d9b72ce9340>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/2.0/mlflow/experiments/get-by-name?experiment_name=bangalore_home_prices","logger":"urllib3.connectionpool"}
{"timestamp":"2025-07-10T13:31:53.992618","level":"warning","event":"Retrying (Retry(total=2, connect=2, read=7, redirect=7, status=7)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7d9b72ce9370>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/2.0/mlflow/experiments/get-by-name?experiment_name=bangalore_home_prices","logger":"urllib3.connectionpool"}
{"timestamp":"2025-07-10T13:32:58.427005","level":"warning","event":"Retrying (Retry(total=1, connect=1, read=7, redirect=7, status=7)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7d9b72ce95e0>: Failed to establish a new connection: [Errno 111] Connection refused')': /api/2.0/mlflow/experiments/get-by-name?experiment_name=bangalore_home_prices","logger":"urllib3.connectionpool"}
{"timestamp":"2025-07-10T13:33:33.547862Z","level":"error","event":"Server indicated the task shouldn't be running anymore. Terminating process","detail":{"detail":{"reason":"not_running","message":"TI is no longer in the running state and task should terminate","current_state":"failed"}},"logger":"task"}
{"timestamp":"2025-07-10T13:33:33.571790Z","level":"error","event":"Task killed!","logger":"task"}
